src/main/antlr/EarthLexer.g4

```diff
@@ -0,0 +1,231 @@
+lexer grammar EarthLexer;
```

Comment:
is there a way for us to still use the same lexer defined in earthly/earthly (or somehow use it as source of truth)?